packages/components/inputs/text-input/src/text-input.tsx

```diff
@@ -61,7 +61,7 @@ type TTextInputProps = {
   /**
    * Horizontal size limit of the input fields.
    */
-  horizontalConstraint:
+  horizontalConstraint?:
```

Comment:
Shouldn't the readme be regenerated?

Reply:
thanks!

Reply:
https://github.com/commercetools/ui-kit/pull/1892/commits/fe2eff259d71680c070f6507dfccfd7d8c3ebc18